src/TextInput/TextInput.jsx

```diff
@@ -90,6 +90,7 @@ export class TextInput extends React.Component {
           required={this.props.required}
           type={type}
           value={this.props.value}
+          {...inputProps}
```

Comment:
it seems a little unusual to have some input properties explicitly listed, while others passed in as `inputProps`

cc @kofi-clever thoughts?